retroarch.c

```diff
@@ -35213,6 +35236,15 @@ static bool retroarch_parse_input_and_config(
       config_load(&p_rarch->g_extern);
    }
 
+   verbosity_enabled = verbosity_is_enabled();
+   /* Init logging after config load only if not overridden by command line argument 
+    * This handles when logging is set in the config but not via the --log-file option. */
+   if (verbosity_enabled && !retroarch_override_setting_is_set(RARCH_OVERRIDE_SETTING_LOG_TO_FILE, NULL))
```

Comment:
This is wrong.

You will call `rarch_log_file_init()` twice if the user sets the command line option `-v` without specifying a log file on the command line...

Reply:
Now `rarch_log_file_init()` only gets called before the config load if the log to file override has been set via the command line switch, so it won't run twice. That said, it seems this function is smart enough to not do anything if it determines no changes in file logging are needed. This makes sense since from the debugging I was doing while testing this, we hit these lines each time content is loaded or unloaded.